website/docs/migration/add_listener/add_listener.dart

```diff
@@ -0,0 +1,16 @@
+// ignore_for_file: avoid_print
+
+import 'package:flutter/material.dart';
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+part 'add_listener.g.dart';
+
+/* SNIPPET START */
+@riverpod
+class MyNotifier extends _$MyNotifier {
+  @override
+  int build() {
+    ref.listenSelf((_, next) => debugPrint('$next'));
```

Comment:
This will be useless since the notifier offers no way to be modified from the UI :)